status/status_test.go

```diff
@@ -119,6 +119,33 @@ func TestFromErrorOK(t *testing.T) {
 	}
 }
 
+func TestFromErrorUnknownError(t *testing.T) {
+	code, message := codes.Unknown, "unknown error"
+	err := errors.New("unknown error")
+	s, ok := FromError(err)
+	if ok || s.Code() != code || s.Message() != message {
+		t.Fatalf("FromError(nil) = %v, %v; want <Code()=%s, Message()=%q, Err=nil>, true", s, ok, code, message)
```

Comment:
Please remove the `Err=nil` and similar below

Reply:
Will do.